forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePINEditText.kt

```diff
@@ -33,6 +40,7 @@ class ForagePINEditText @JvmOverloads constructor(
         }
         addView(vault!!.getUnderlying())
         addView(getLogoImageViewLayout(context))
+        logger.i("ForagePINEditText successfully rendered")
```

Comment:
include `vault` in log? Just because the value is there it could be helpful. May not be necessary though..

Reply:
This is now captured in the `LDManager`